README.md

```diff
@@ -91,6 +91,10 @@ Assignment is denoted by the `put/into` keyword combination:
 * `Put 123 into X` will assign the value `123` to the variable `X`
 * `Put "Hello San Francisco" into the message` will assign the value `"Hello San Francisco"` to the variable `the message`
 
+`thinks` is a valid alias for the `put/into` combination and allows full subexpressions.
+
+* `My brother thinks my sister is smaller than My Other Brother` will assign the variable `My brother` to the result of `my sister < My Other Brother`.
```

Comment:
Shouldn't this be `my other brother`?

Reply:
I believe the intent is that `My Other Brother` is a proper variable - `my other brother` would not be a valid common variable since common variables can't have spaces. However, given the recent discussion at #148 I would say that `My Other Brother` isn't a valid proper variable either since it uses "My", unless "My" isn't considered a keyword?

Reply:
I think #117 is more appropriate, but yeah, my bad, it should be okay indeed.

Reply:
That was my intent, though clarity on the `My` debate should be cleared in #148 first.